design-system/scripts/generate-custom-properties.js

```diff
@@ -31,29 +31,35 @@ const allThemesNames = Object.keys(definitions.choiceGroupsPerTheme);
 const tokens = {};
 const designTokens = {};
 
-Object.entries(definitions.choiceGroupsPerTheme).forEach(
-  ([themeName, themeChoiceGroups]) => {
-    if (!tokens[themeName]) {
-      tokens[themeName] = {};
-    }
-    Object.values(themeChoiceGroups).forEach((themeChoiceGroup) => {
-      Object.entries(themeChoiceGroup.choices).forEach(([key, value]) => {
-        if (tokens[themeName][key])
-          endProgram(`Token "${key} already exists!"`);
-
-        if (key !== key.toLowerCase())
-          endProgram(`Tokens "${key}" must be lower case`);
-
-        if (!key.startsWith(themeChoiceGroup.prefix))
-          endProgram(
-            `Expected token "${key}" to start with "${themeChoiceGroup.prefix}" as it is an "${themeChoiceGroup.label}" attribute.`
-          );
+const defaultTheme = definitions.choiceGroupsPerTheme.default;
 
-        tokens[themeName][key] = value;
-      });
-    });
+Object.keys(definitions.choiceGroupsPerTheme).forEach((themeName) => {
```

Comment:
It seems we're recreating the default theme in the `customProperties` file for alternative themes. 
Perhaps we could just have the overwritten properties and always merge from `default` theme in the `ThemeProvider`.
What do you think?